src/storage/index.ts

```diff
@@ -260,6 +267,10 @@ async function getFileContents(path: string, app: string, username: string | und
     throw await getBlockstackErrorFromResponse(response, `getFile ${path} failed.`)
   }
   const contentType = response.headers.get('Content-Type')
+  const etag = JSON.parse(response.headers.get('ETag'))
```

Comment:
Could the etag ever be corrupt/unparsable and cause `JSON.parse` to throw an error?

Reply:
Was using `JSON.parse()` to get rid of redundant double quotes (for some reason the storage driver was returning etags with nested quotes). Replacing this with `replace(/^"|"$/g, '')` as per Matt's suggestion.